src/frontend/Dockerfile

```diff
@@ -8,15 +8,25 @@ COPY package*.json ./
 FROM base as dev
 ENV NODE_ENV=development
 ENV BUILD_PATH=build
-RUN npm ci --verbose --production=false && npm cache clean --force
+RUN npm i --verbose
```

Comment:
I think day to day development needs to use `npm i` in order to prevent npm from deleting the local filesystem's `node_modules`